packages/lime-system/files/usr/lib/lua/lime/generic_config.lua

```diff
@@ -17,10 +19,10 @@ function gen_cfg.configure()
     gen_cfg.do_generic_uci_configs()
     gen_cfg.do_copy_assets()
     if not utils.file_exists(gen_cfg.CONFIG_FIRST_BOOT_SIGNAL_FILE) then
-        gen_cfg.do_run_assets('FIRSTBOOT')
+        gen_cfg.do_run_assets(gen_cfg.RUN_ASSET_FIRSTBOOT)
         utils.write_file(gen_cfg.CONFIG_FIRST_BOOT_SIGNAL_FILE, '')
     end
-    gen_cfg.do_run_assets('RECONFIG')
+    gen_cfg.do_run_assets(gen_cfg.RUN_ASSET_AT_CONFIG)
```

Comment:
RUN_ASSET_AT_CONFIG uses AT, and RUN_ASSET_FIRSTBOOT not, please fix using AT in both or in none.

Reply:
Thanks! Fixed